packages/cli/src/lib/axe-test-urls.ts

```diff
@@ -1,42 +1,44 @@
-'use strict';
+import * as WebDriver from 'selenium-webdriver';
+// @ts-ignore
```

Comment:
Why?

Reply:
Have to wait for this PR to land to remove the `@ts-ignore` https://github.com/dequelabs/axe-core-npm/pull/197